src/Forms/UrlField.php

```diff
@@ -7,26 +7,89 @@
 
 /**
  * Text input field with validation for a url
- * Url must include a scheme, either http:// or https://
+ * Url must include a protocol (aka scheme) such as https:// or http://
  */
 class UrlField extends TextField
 {
+    /**
+     * The default set of protocols allowed for valid URLs
+     */
+    private static array $default_protocols = ['https', 'http'];
+
+    /**
+     * The default value for whether a relative protocol (// on its own) is allowed
+     */
+    private static bool $default_allow_relative_protocol = false;
```

Comment:
Added the ability to define globally whether `//` is okay instead of a specific protocol